src/create_update.c

```diff
@@ -387,8 +387,10 @@ int main(int argc, char **argv)
 		name_includes = manifest->includes;
 		while (name_includes) {
 			char *name = name_includes->data;
+			gpointer tmp = g_hash_table_lookup(new_manifests, name);
```

Comment:
@ikeydoherty:
I would have rather thrown away glib in the name of clarity :-P
My patch simply tried to go with the flow of the code, but I have no qualm to do as you suggest.

However I'd appreciate if I could get a final statement on this: is this patch something that the swupd team would consider merging, even as temporary workaround, or should we carry it on as Ostro only?